homeassistant/components/smarttub/config_flow.py

```diff
@@ -24,30 +19,73 @@ class SmartTubConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
 
+    def __init__(self) -> None:
+        """Instantiate config flow."""
+        super().__init__()
+        self._reauth_input = None
+        self._reauth_entry = None
+
     async def async_step_user(self, user_input=None):
         """Handle a flow initiated by the user."""
         errors = {}
 
-        if user_input is None:
-            return self.async_show_form(
-                step_id="user", data_schema=DATA_SCHEMA, errors=errors
-            )
+        data_schema = vol.Schema(
+            {vol.Required(CONF_EMAIL): str, vol.Required(CONF_PASSWORD): str}
+        )
```

Comment:
Slightly cleaner to leave this as the constant since you only need to inject the default in `async_step_reauth_confirm`

Reply:
:shrug: it's now only used in the one method. but I can pull it out, sure